src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
+
+async function documenter(options = {}) {
+  // print schemas
+  console.log(options.schemas);
+
+  // take schemas(name + content) + add it to stream tarball
+  // pack is a streams2 stream
+  let tarball = tar.pack();
+
+  let schemas = options.schemas;
+
+  if (schemas) {
+    for (let schema of schemas) {
+      tarball.entry({name: schema.name}, schema.content);
+    }
+  }
+  tarball.finalize();
+
+  let output = {
+    tarball,
+  };
+
+  return output;
+
+  // add docs to tarball
```

Comment:
Docs should already have filenames...
So something like:
```js
let fs = require('mz/fs');

let files = await listFolderRecursively(options.folder); // List all files in the folder
files = files.map(file => path.relative(file, options.folder)); // Convert file path to relative paths
await Promise.all(files.map(async (file) => {
  let data = await fs.readFile(path.join(options.folder, file));
  tarball.entry({name: 'docs/' + file}, data);
}));
```

Now a file at `<options.folder>/mydocs.md` will be `docs/mydocs.md` inside the tarball..